rllib/BUILD

```diff
@@ -379,6 +379,17 @@ py_test(
     args = ["--yaml-dir=tuned_examples/impala"]
 )
 
+# MADDPG
+py_test(
+    name = "learning_tests_two_step_game_maddpg",
+    main = "tests/run_regression_tests.py",
+    tags = ["team:ml", "tf_only", "no_tf_eager_tracing", "learning_tests", "learning_tests_discrete"],
```

Comment:
Could we maybe instead add this to the weekly ci -- I feel like we don't need to as widely support this algorithm until there is a customer use case.

Reply:
Good question: I think we should move toward:
* CI-tests: small tasks that can be learnt in 1-2 min (CartPole, **TwoStepGame**, maybe Pendulum or simpler cont. actions problem)
* nightly tests: hard tasks, requiring GPU + ~1h of completion on >1 workers

Reply:
I'll leave this test here for now to roughly match the other algos' coverage (everyone has CartPole/Pendulum tests in the CI, which is ok).